README

```diff
@@ -71,7 +71,7 @@ Currently Net::DRI ships with:
     * https://tools.ietf.org/html/draft-wang-eppext-domain-verification-01
 - a full DAS (Domain Availability Service) implementation for .BE .EU .NL .AU
 - a full Whois implementation (RFC 3912) for thin and thick registries :
-     support included for domain names in .COM .NET .ORG .BIZ .INFO .AERO
```

Comment:
Can you revert the changes in this file please, I will review them at a later date. In this case, it does support .BIZ ... just in a different way.